File: LENTABP10/src/main/java/com/lenta/lentabp10/MobileObjectsService/models/MB_S_34.java
package com.lenta.lentabp10.MobileObjectsService.models;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;

// Модель справочника настроек заданий на списание WOB (справочник видов товара)
public class MB_S_34 {
    /// Тип задания на списание
    @JsonProperty("TASK_TYPE")
    @Getter private String taskType;

    /// Вид товара
    @JsonProperty("MTART")
    @Getter private String mtart;

    public MB_S_34(String taskType, String mtart) {
        this.taskType = taskType;
        this.mtart = mtart;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/mobileObjectsService/models/MB_S_34.kt
package com.lenta.bp10.mobileObjectsService.models

import com.google.gson.annotations.SerializedName

// Модель справочника настроек заданий на списание WOB (справочник видов товара)
class MB_S_34(/// Тип задания на списание
        @SerializedName("TASK_TYPE")
        val taskType: String,

        /// Вид товара
        @SerializedName("MTART")
        val mtart: String
)